samples/dialogs/src/main/java/com/badoo/ribs/samples/dialogs/rib/dialogs_example/DialogsPresenter.kt

```diff
@@ -26,18 +29,16 @@ interface DialogsPresenter {
 internal class DialogsPresenterImpl(
     private val dialogs: Dialogs,
     private val backStack: BackStack<DialogsRouter.Configuration>
-) : DialogsPresenter, ViewAware<DialogsView> {
+) : DialogsPresenter, ViewAware<DialogsView>, SubtreeChangeAware {
 
     private var view: DialogsView? = null
```

Comment:
Don't store references to view:

1. it can leak
2. in other methods you need null checks and assume it's there